packages/domainproperties/src/components/assay/AssayPropertiesInput.tsx

```diff
@@ -53,7 +82,6 @@ export function NameInput(props: InputProps) {
                 return (
                     <>
                         <p>The name for this assay design. Note that this can't be changed after the assay design is created.</p>
-                        <p><small><i>This field is required.</i></small></p>
```

Comment:
Was there a reason this "required" text was removed from the tooltip here? Maybe it was supposed to come from the prop being set as `required={true}`, but I am not seeing it in the UI for the assay prop name tooltip.